x-pack/legacy/plugins/lens/public/editor_frame_plugin/merge_tables.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ExpressionFunction } from 'src/legacy/core_plugins/interpreter/types';
+import { LensMultiTable, KibanaDatatable } from '../types';
+
+interface MergeTables {
+  layerIds: string[];
+  tables: KibanaDatatable[];
+}
+
+export const mergeTables: ExpressionFunction<
+  'lens_merge_tables',
+  null,
+  MergeTables,
+  LensMultiTable
+> = {
+  name: 'lens_merge_tables',
+  type: 'lens_multitable',
+  help: i18n.translate('xpack.lens.functions.mergeTables.help', {
+    defaultMessage: 'A helper to merge any number of kibana tables into a single table',
```

Comment:
This help text is a bit misleading. It merges an array of tables into a `{ tables: Dictionary<string, Table> }` object...